zone/inventory.cpp

```diff
@@ -1963,7 +1962,7 @@ bool Client::SwapItem(MoveItem_Struct* move_in) {
 	if (((with && with->IsClient() && dst_slot_id >= EQ::invslot::TRADE_BEGIN && dst_slot_id <= EQ::invslot::TRADE_END) ||
 		(dst_slot_id >= EQ::invslot::SHARED_BANK_BEGIN && dst_slot_id <= EQ::invbag::SHARED_BANK_BAGS_END))
 	&& GetInv().CheckNoDrop(src_slot_id)
-	&& RuleI(World, FVNoDropFlag) == 0 || RuleI(Character, MinStatusForNoDropExemptions) < Admin() && RuleI(World, FVNoDropFlag) == 2) {
+	&& !EQ::inventory::CanTradeNoDropItem(Admin())) {
```

Comment:
A note: this check was originally incorrect and an admin would get flagged for trying to trade an item here.